src/util/saveArticles.ts

```diff
@@ -402,7 +422,25 @@ async function treatVideo(mw: MediaWiki, dump: Dump, srcCache: KVS<boolean>, art
     }
 
     sourceEl.setAttribute('src', newUrl);
-    return { mediaDependencies };
+
+    /* Scrape subtitle */
+    const trackEle = videoEl.querySelector('track');
```

Comment:
It looks like you assume here there is only one subtitle?! It should obviously work (and be tested) with many.